packages/waku/src/vite-rsc/plugin.ts

```diff
@@ -14,16 +13,14 @@ import path from 'node:path';
 import assert from 'node:assert/strict';
 import fs from 'node:fs';
 import type { Config } from '../config.js';
-import { INTERNAL_setAllEnv, unstable_getBuildOptions } from '../server.js';
+import { INTERNAL_setAllEnv } from '../server.js';
 import { emitStaticFile, waitForTasks } from '../lib/builder/build.js';
-import {
-  getManagedEntries,
-  getManagedMain,
-} from '../lib/plugins/vite-plugin-rsc-managed.js';
+import { getManagedMain } from '../lib/plugins/vite-plugin-rsc-managed.js';
```

Comment:
Is it impossible to keep `getManagedEntries` and `getManagedMain` outside of this file? It can be something like `packages/waku/src/lib/managed.ts`.

Reply:
I don't see much files under `packages/waku/src/lib`. Maybe move it to `packages/waku/src/lib/utils`?